### PR TITLE
Feature/tables mobile view

### PR DESCRIPTION
**Pull request by monkin77**

### CODE REVIEW

**Comment by monkin77** [comment]
image
image

**Comment by monkin77** [comment]
> Lots of extra commits eyes
> 
> Apart from that, is the table footer having a horizontal scroll issue? Looks like something's missing when looking at the screenshots

I'll take care of that :sweat_smile: . Regarding the footer, I've already fixed it but the screenshots aren't updated

File: src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js [hunk 1]
```diff
@@ -56,7 +57,7 @@ const filters = [
     { id: "location-filter", render: LocationFilter },
 ];
 
-const RowActions = ({ row }) => {
+/* const RowActions = ({ row }) => {
```
Comment (TiagooGomess) [inline src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js:60]: Remove commented code.

File: src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js [hunk 2]
```diff
@@ -125,6 +129,73 @@ const CompanyOffersManagementWidget = ({ addSnackbar }) => {
...
+    const RowCollapseComponent = ({ rowKey }) => {
+        const row = offers[rowKey];
+        const offerRoute = `/offer/${rowKey}`;
+        const classes = useRowCollapseStyles();
+        const mobileFieldKeys = ["location", "publishEndDate"];
+
+        return (
+            !isMobile ? (
+                <div />
+            ) : (
```
Comment (TiagooGomess) [inline src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js:156]:
```suggestion
            isMobile && (
```

**Review by TiagooGomess — CHANGES_REQUESTED** [review]

**Review by TiagooGomess — APPROVED** [review]

**Pull request merged**